wazo_call_logd/cel_interpretor.py

```diff
@@ -8,11 +8,17 @@
 from xivo.asterisk.line_identity import identity_from_channel
 
 from .database.cel_event_type import CELEventType
-from .database.models import CallLogParticipant, Recording
+from .database.models import CallLogParticipant, Destination, Recording
 
 
 logger = logging.getLogger(__name__)
 
+EXTRA_WAZO_CALL_LOG_DESTINATION_UUID_REGEX = (
+    r'^.*type: *(.*?) *, *uuid: *(.*?) *, *name: *(.*?) *(?:,|"})'
+)
+EXTRA_WAZO_CALL_LOG_DESTINATION_CONFERENCE_REGEX = (
+    r'^.*type: *(.*?) *,*id: *(.*?) *(?:,|"})'
+)
```

Comment:
Can you make it more generic, so that the regex can take an arbitrary key-value set? That would allow to share the same regex for all destinations, and so that wazo-call-logd is not dependent on the order of parameters.

Reply:
In the end I went back to a simple and effective/generic solution (in my opinion); by converting the string to key-value dict.